mytoyota/client.py

```diff
@@ -455,3 +456,87 @@ async def get_driving_statistics_json(
         return json.dumps(
             await self.get_driving_statistics(vin, interval, from_date), indent=3
         )
+
+    async def get_trips(self, vin: str) -> list(Trip):
+        """Returns a list of trips.
+
+        Retrieves and formats trips.
+
+        Args:
+            vin (str):
+                Vehicle identification number.
+
+        Returns:
+            A list of trips.
+
+        Raises:
+            ToyotaLoginError: An error returned when updating token or invalid login information.
+            ToyotaInternalError: An error occurred when making a request.
+            ToyotaApiError: Toyota's API returned an error.
+        """
+        _LOGGER.debug(f"Getting trips for {censor_vin(vin)}...")
+
+        raw_trips = await self.api.get_trips_endpoint(vin)
+        _LOGGER.debug(f"received {len(raw_trips.get('recentTrips',[]))} trips")
+        return [Trip(trip) for trip in raw_trips]

```

Comment:
The `raw_trips` dictionary has a 'recentTrips' value, which is the actual list of trips. Shouldn't the content of that value be used when creating `Trip` objects? That 'recentTrips' value is also provided to the debug-logger....

Reply:
Gotcha. Figured I'd add some more tests as well, and also converted the timestamps into python's datetime to make it easier to compare with date/time objects https://github.com/DurgNomis-drol/mytoyota/pull/188/commits/9ddcb7ccc471c6750c450972068601d534f2b50c